uaaextras/templates/maintenance.html

```diff
@@ -0,0 +1,11 @@
+{% extends "_layout.html" %}
+
+{% block header %}System Maintenance{% endblock %}
+
+{% block content %}
+<div class="island-content">
+    <p>Our system is currently undergoing scheduled maintenance and will be back online soon.</p>
```

Comment:
of the options discussed, this really sounds like the hardest and worst experience for users to me

Reply:
Agreed after our discussion yesterday, but this might be useful to have just in case for the future.